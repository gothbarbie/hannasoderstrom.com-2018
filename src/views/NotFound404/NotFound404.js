import React from 'react'
import { Link } from 'react-router-dom'

import Wrapper from '../../components/Layout/Wrapper'
import H1 from '../../components/H1'
import Paragraph from '../../components/Paragraph'

const NotFound404 = () => (
  <Wrapper column>
    <H1>Not Found</H1>
    <Paragraph>We could not find what you were looking for.</Paragraph>
    <Paragraph>
      <Link to="/">‚Üê Back to Home Page</Link>
    </Paragraph>
  </Wrapper>
)

export default NotFound404
