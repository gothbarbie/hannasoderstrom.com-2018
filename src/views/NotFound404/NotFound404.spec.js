import React from 'react'
import { shallow } from 'enzyme'

import NotFound404 from './index'

let component

describe('views/NotFound404', () => {
  beforeEach(() => {
    component = shallow(<NotFound404 />)
  })

  it('renders', () => {
    expect(component).toMatchSnapshot()
  })
})
