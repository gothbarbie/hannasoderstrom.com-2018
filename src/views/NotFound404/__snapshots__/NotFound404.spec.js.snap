// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/NotFound404 renders 1`] = `
<styled.div
  column={true}
>
  <styled.h1>
    Not Found
  </styled.h1>
  <styled.p>
    We could not find what you were looking for.
  </styled.p>
  <styled.p>
    <Link
      replace={false}
      to="/"
    >
      ‚Üê Back to Home Page
    </Link>
  </styled.p>
</styled.div>
`;
